### DISCUSSION TITLE
Stronghold

### DISCUSSION THREAD

**Discussion by magnitood** [Zodsmar/SeedSearcherStandaloneTool#85] [General]
Is stronghold searching possible to add?

**Comment by Zodsmar** [comment]
https://github.com/Zodsmar/SeedSearcherStandaloneTool/wiki/Stronghold-Searching this is why it won't be added

**Reply by magnitood** [reply]
Ok that explains thanks

**Reply by Zodsmar** [reply]
In 1.18 I will be adding the feature to search strongholds. Even though these limits still apply, you can search for strongholds, within reason. But the feature will be there

**Reply by magnitood** [reply]
Great thanks, appreciate it
